mlir/test/Dialect/Tosa/canonicalize.mlir

```diff
@@ -760,7 +760,7 @@ func.func nested @do_not_fold_reciprocal_int() -> tensor<3x600x1200xi32> {
 
 // CHECK-LABEL: @canonicalize_select_to_clamp
 func.func @canonicalize_select_to_clamp(%arg0: tensor<13x21x3xf32>) -> tensor<13x21x3xf32> {
-// CHECK:           %[[VAL_1:.*]] = tosa.clamp %arg{{.*}} {max_fp = 0x7F800000 : f32, max_int = 9223372036854775807 : i64, min_fp = 1.500000e+00 : f32, min_int = -9223372036854775808 : i64} : (tensor<13x21x3xf32>) -> tensor<13x21x3xf32>
+// CHECK:           %[[VAL_1:.*]] = tosa.clamp %arg{{.*}} {max_fp = 3.40282347E+38 : f32, max_int = 9223372036854775807 : i64, min_fp = 1.500000e+00 : f32, min_int = -9223372036854775808 : i64} : (tensor<13x21x3xf32>) -> tensor<13x21x3xf32>
```

Comment:
Why is this correct?
The original code will return `inf` when `%arg0` is `inf`, but the new tosa.clamp op would instead return `3.40282347E+38`.